notsobot/notsobot.py

```diff
@@ -1623,12 +1718,16 @@ async def rotate(self, ctx, degrees: int = 90, urls: ImageFinder = None):
 
             def rotate_img(b, degrees):
                 with Image.open(b).convert("RGBA") as img:
-                    img = img.rotate(int(degrees))
+                    image = img.rotate(int(degrees))
+                    img.close()
```

Comment:
This is unnecessary since `img` was created by the context manager which closes it automatically.